<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginDlg</class>
 <widget class="QDialog" name="CLoginDlg">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CLoginDlg</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TQLive.qrc">
    <normaloff>:/TQLive/images/logo.png</normaloff>:/TQLive/images/logo.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgba(47,47,47,1);
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CCustomHeadBar" name="chb_head_bar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgba(47,47,47,1);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_G">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_F">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>244</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cb_close_window">
        <property name="minimumSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
border: noborder;
background-image: url(:/TQLive/images/pb_close_normal.png);
}
QPushButton:hover{
background-image: url(:/TQLive/images/pb_close_pressed.png);
}
QPushButton:pressed{
background-image: url(:/TQLive/images/pb_close_pressed.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>31</number>
     </property>
     <property name="bottomMargin">
      <number>18</number>
     </property>
     <item>
      <widget class="QLabel" name="lb_application_logo">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">margin:0px 0px 0px 0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="TQLive.qrc">:/TQLive/images/logo.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_application_name">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:20px;
font-family:&quot;微软雅黑&quot;;
font-weight:500;
color:rgba(220,220,220,1);
margin:0px 0px 0px 8px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="TQLive.qrc">:/TQLive/images/lb_application_name.png</pixmap>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_no_account">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>48</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">margin:0px 0px 0px 43px;
font-size:12px;
font-family:&quot;微软雅黑&quot;;
font-weight:400;
color:rgba(119,119,119,1);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="TQLive.qrc">:/TQLive/images/lb_no_account.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_register">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font-size:12px;
font-family:&quot;微软雅黑&quot;;
color:rgba(74,177,52,1);
background:transparent;
font-weight:500;
padding-left:0px;
padding-right:0px;
}
QPushButton:hover,QPushButton:pressed{
color:#DCDCDC;
}</string>
       </property>
       <property name="text">
        <string>马上注册</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>26</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="TQLive.qrc">:/TQLive/images/lb_QR_login.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_A">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgba(63,63,63,1);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="CCustomLabel" name="lb_phone_login">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>55</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:16px;
font-family:&quot;微软雅黑&quot;;
font-weight:500;
color:rgba(220,220,220,1);
margin-top: 18px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="TQLive.qrc">:/TQLive/images/lb_phone_login_pressed.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>8</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1</width>
         <height>8</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgba(63,63,63,1);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CCustomLabel" name="lb_account_login">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>55</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:16px;
font-family:&quot;微软雅黑&quot;;
font-weight:400;
color:rgba(74,177,52,1);
margin-top: 18px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="TQLive.qrc">:/TQLive/images/lb_account_login_normal.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QStackedWidget" name="sw_login_method">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>340</width>
         <height>142</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>340</width>
         <height>142</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="CPhoneLoginWidget" name="page_3"/>
       <widget class="CAccountLoginWidget" name="page_4"/>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_D">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>43</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>11</number>
       </property>
       <item>
        <widget class="QPushButton" name="pb_login">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>340</width>
           <height>46</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>340</width>
           <height>46</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
font-size:18px;
font-family:&quot;微软雅黑&quot;;
font-weight:400;
color:rgba(255,255,255,1);
border: noborder;
color: white;
background-image: url(:/TQLive/images/pb_common_normal.png);
}
QPushButton:hover{
background-image: url(:/TQLive/images/pb_common_hover.png);
}
QPushButton:pressed{
background-image: url(:/TQLive/images/pb_common_pressed.png);
}</string>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lb_prompt_msg">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>276</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>276</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>提示信息</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QPushButton" name="btn_forget_passwd">
           <property name="styleSheet">
            <string notr="true">QPushButton{
font-size:12px;
font-family:&quot;微软雅黑&quot;;
font-weight:400;
color:rgba(255,140,5,1);
background:transparent;
padding-left:0px;
padding-right:0px;
}
QPushButton:hover,QPushButton:pressed{
color:#DCDCDC;
}</string>
           </property>
           <property name="text">
            <string>忘记密码？</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_C">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>43</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="Line" name="line">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>1</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgba(63,63,63,1);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>67</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>67</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:16px;
font-family:&quot;微软雅黑&quot;;
font-weight:400;
color:rgba(153,153,153,1);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="TQLive.qrc">:/TQLive/images/lb_third_party_login.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>1</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgba(63,63,63,1);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>87</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>87</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">margin:0px 0px 0px 73px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="TQLive.qrc">:/TQLive/images/lb_wechat_login.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>87</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>87</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">margin:0px 73px 0px 0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="TQLive.qrc">:/TQLive/images/lb_qq_login.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CCustomHeadBar</class>
   <extends>QWidget</extends>
   <header>ccustomheadbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CCustomLabel</class>
   <extends>QLabel</extends>
   <header location="global">ccustomlabel.h</header>
  </customwidget>
  <customwidget>
   <class>CPhoneLoginWidget</class>
   <extends>QWidget</extends>
   <header>cphoneloginwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CAccountLoginWidget</class>
   <extends>QWidget</extends>
   <header location="global">caccountloginwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
